test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -1854,3 +1717,64 @@ Describe "Web cmdlets tests using the cmdlet's aliases" -Tags "CI" {
         $result.Hello | Should Be "world"
     }
 }
+
+Describe -Name "Web cmdlets tests using Client Certificate Authentication" -Tags "Feature","Slow" {
+
+    BeforeAll { 
+        $portNumber          = 8443
+        $uri                 = "https://localhost:{0}/" -f $portNumber
+        $containerName       = 'certcheck'
+        $initTimeoutSeconds  = 15
+        $containerPath       = Join-Path $PSScriptRoot 'assets\docker\ClientCertificateCheck'
+        $pfxPath             = Join-Path  $containerPath 'ClientCert.pfx'
+        $initCompleteMessage = 'Now listening on'
+        $successMessage      = 'Status: OK'
+        $failedMessage       = 'Status: Failed'
+        
+        Push-Location $containerPath
+        $null = docker build -t $containerName .
+        $timeOut = (get-date).AddSeconds($initTimeoutSeconds)
+        $null = docker run -d -p ${portNumber}:${portNumber} --name $containerName $containerName $portNumber
```

Comment:
If we compile the container at CI start, that couples the test with an entire build scenario instead of a test scenario.